src/Reflection/ReflectionMethod.php

```diff
@@ -93,13 +85,10 @@ public function __wakeup()
         $this->reflection = new \ReflectionMethod($this->classReflection->getName(), $this->name);
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    protected function reflect()
+    protected function reflect(): void
     {
         $docComment = $this->reflection->getDocComment();
-        if (strpos($docComment, self::INHERIT_TAG) !== false) {
+        if (false !== $docComment && strpos($docComment, self::INHERIT_TAG) !== false) {
```

Comment:
is it some kind of bugfix here?

Reply:
Yes. `reflectionMethod()` possibly can return a non-string value (`false`), which causes a type error.

Reply:
The bug is only present with strict typing, which is why it never occured in `2.0`. IMO there is not bugfix needed for previous version, where strict typing is not supported with PHP ^5.0.